src/components/04-modules/grid/grid.scss

```diff
@@ -2,26 +2,29 @@
   @include wrapper;
 
 
-  @include breakpoint($medium) { 
+  @include breakpoint($medium) {
     display: grid;
     grid-gap: 16px;
     grid-auto-rows: auto;
     grid-auto-flow: row;
-    grid-gap: 32px; 
+    grid-gap: 32px;
   }
 
-  &--two-columns { 
+  &--two-columns {
     @include breakpoint($medium) { grid-template-columns: repeat(2, 1fr); }
   }
-  &--three-columns { 
+  &--three-columns {
     @include breakpoint($medium) { grid-template-columns: repeat(3, 1fr); }
   }
-  &--four-columns { 
+  &--four-columns {
     @include breakpoint($medium) { grid-template-columns: repeat(4, 1fr); }
   }
+  &--five-columns {
+    @include breakpoint($medium) { grid-template-columns: repeat(5, 1fr); }
+  }
 
   &__2 { grid-column: auto / span 2; }
   &__3 { grid-column: auto / span 3; }
+  &__5 { grid-column: auto / span 5; }
```

Comment:
How come there is no `&__5 { grid-column: auto / span 4; }`?

Reply:
Not sure! apparently there never was one, I'm not entirely sure what it affects so I just added it on the basis of there being an entry for 2 and 3.